packages/create-playwright/assets/examples/3-assertions.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+
+// For a full overview see here: https://playwright.dev/docs/test-assertions/
+test('should be able to use assertions', async ({ page, baseURL }) => {
+  await page.goto('/assertions.html');
+
+  // Useful for input[type='checkbox']
+  await expect(page.locator('#my-checked-checkbox')).toBeChecked();
+
+  // Useful for input elements
+  await expect(page.locator('#my-input-with-value')).toHaveValue('foobar');
+  await expect(page.locator('#my-empty-input')).toBeEmpty();
+
+  await expect(page.locator('#my-paragraph')).toContainText('Dorem');
+  await expect(page.locator('#my-paragraph')).toHaveText('Lorem Dorem Ipsum');
```

Comment:
Add an example where we match an array of items, and an example with RegExp object.